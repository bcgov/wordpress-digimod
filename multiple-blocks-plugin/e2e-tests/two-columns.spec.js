import {
    createNewPost,
    enablePageDialogAccept,
    insertBlock,
    getEditedPostContent
} from '@wordpress/e2e-test-utils';

describe( 'Two columns block', () => {
    jest.setTimeout(60000);

    beforeAll( async () => {
        await enablePageDialogAccept();
    } );
    beforeEach( async () => {
        await createNewPost();
    } );

    it( 'Two columns block should be available', async () => {
        await insertBlock( 'two-columns' );
		// Check if block was inserted
        
		expect( await page.$( '[data-type="multiple-blocks-plugin/two-columns"]' ) ).not.toBeNull();

		expect( await getEditedPostContent() ).toMatchSnapshot();
	} );
} );