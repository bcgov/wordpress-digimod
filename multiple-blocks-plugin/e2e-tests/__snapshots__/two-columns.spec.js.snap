// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Two columns block Two columns block should be available 1`] = `
"<!-- wp:multiple-blocks-plugin/two-columns -->
<div class=\\"row\\"></div>
<!-- /wp:multiple-blocks-plugin/two-columns -->"
`;
